src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -134,15 +134,18 @@ public static class Blocks {
          */
         public static final TagKey<Block> ORE_BEARING_GROUND_STONE = tag("ore_bearing_ground/stone");
         /**
-         * Ores which on average result in more than one resource worth of materials
+         * Ores which on average result in more than one resource worth of materials ignoring fortune and other modifiers.
+         * (example, Redstone Ore)
```

Comment:
The prime example that needs to be listed here is copper ore, its the reason this tag was made, to distinguish it from other metals